cmd/namespace/migrate_down.go

```diff
@@ -1,56 +1,21 @@
 package namespace
 
 import (
-	"fmt"
-	"strconv"
-
 	"github.com/ory/keto/cmd/migrate"
 
-	"github.com/ory/x/cmdx"
 	"github.com/spf13/cobra"
-
-	"github.com/ory/keto/internal/driver"
 )
 
 func NewMigrateDownCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "down <namespace-name> <steps>",
-		Short: "Migrate a namespace down",
+		Deprecated: "This step is not necessary anymore, see https://github.com/ory/keto/pull/638",
```

Comment:
I think it's fine to delete this command

Reply:
If we instead we add the deprecated text, someone using it will now what is going on. We can delete it at a later point. A deprecated command does not show up in the help or docs.

Reply:
True